manager.go

```diff
@@ -1187,6 +1185,12 @@ func (m *Manager) activateProbes() {
 
 // UpdateActivatedProbes - update the list of activated probes
 func (m *Manager) UpdateActivatedProbes(selectors []ProbesSelector) error {
+	m.stateLock.Lock()
```

Comment:
Where do you unlock this mutex if the next if branch is not taken ?

Reply:
Happy copy pasting, thanks for catching it :) fixed 🐌